x-pack/plugins/osquery/server/search_strategy/osquery/index.ts

```diff
@@ -39,7 +39,7 @@ export const osquerySearchStrategyProvider = <T extends FactoryQueryTypes>(
           index: `${ACTIONS_INDEX}*`,
         }),
         newDataStreamIndexExists: esClient.asInternalUser.indices.exists({
-          index: `${ACTION_RESPONSES_DATA_STREAM_INDEX}*`,
+          index: `${ACTION_RESPONSES_DATA_STREAM_INDEX}-default`,
```

Comment:
@patrykkopycinski @tomsonpl 
With `*` `exists` api always returns true, with a literal name it returns as expected.